currency/storage.go

```diff
@@ -369,37 +369,42 @@ func (s *Storage) WriteCurrencyDataToFile(path string, mainUpdate bool) error {
 }
 
 // LoadFileCurrencyData loads currencies into the currency codes
-func (s *Storage) LoadFileCurrencyData(f File) error {
+func (s *Storage) LoadFileCurrencyData(f *File) error {
 	for _, contract := range f.Contracts {
-		err := s.currencyCodes.LoadItem(contract)
+		contract := contract
```

Comment:
Perhaps c := &contract and use it param for LoadItem would be better than this double assignment?